es/tabs/offering.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: offerings
+lng_pair: id_offering
+---
+<script>
+    function showContent(index){
+        var servicesItems = document.getElementsByClassName("services-item");
+            var servicesItems = document.getElementsByClassName("services-item");
+            servicesItems[index].classList.toggle("active");
```

Comment:
3 items to fix here:
1. Remove copied line
2. Fix indentation
3. Change `var`  by `const`